modules/sync/bufbuild/protovalidate/cas/91a34278888085531c7c6679b921d49d5487ac191a0ec6e5b9a551b09f4e4722d7ada755a39086c8fc8d8afd1bb2c60103c507b0484a974c203af99e63b94c60

```diff
@@ -0,0 +1,6 @@
+shake256:8f88503768991dcc77b5a39a4a8d3374c99f4f0f69fb9e58a383fa238b20645190dc4570817b8ab9bbd83e90d063d151048137ef8c6bdb51ea797f7c6f41cfba  LICENSE
+shake256:cb645521026366a892de82a61b1a05052160416b923743cc4d92c27c47a95c62d0b8aa0bd15ee27818838f82b281edcd14244ca99e7142f6c2ef2bc132983549  README.md
+shake256:b8ec4cbd3f0ff7194a4c6564eecfecc967ce2a93caeb5a09f402efb172371655b358d7bcb6fd7d6a01c1c2f71f705c3f21a55f159c779ea246f99e385a7f4bbb  buf.yaml
+shake256:7f79103c64037f89731fa57fb8fac46aa080c122d6ee433b94afb3b991b0147ee428b62c650bec03cf6dbb1830dd3bc89add1d7592ce411c54c77a64c52d3a54  buf/validate/expression.proto
+shake256:3f47d5fe276096376f3dc55357ea673924901ba6e753a84710cf37080434668def5b423b19b10d00c1cd42f647e7826bdff46bb88802a8dfae7fc74982137965  buf/validate/priv/private.proto
+shake256:cc2e1b52f09921805a68f93f1f9433b231ac098c4b849f8a08d674113a33c6e95992533de76083eb0c5caf94f3b37c36293549104fd50fdd66ab9c0602473a60  buf/validate/validate.proto
```

Comment:
@rodaine Is `priv/private.proto` supposed to be included?

Reply:
Yes